emperor/support_files/js/sceneplotview3d.js

```diff
@@ -50,7 +50,10 @@ define([
    *
    **/
   ScenePlotView3D = function(renderer, decViews, container, xView, yView,
-                             width, height){
+                             width, height, callback){
+    var scope = this;
+    // Make setting a callback optional
+    this.callback = callback || function(name, item) {};
```

Comment:
The problem with the callback being an attribute, is that then we can only have one subscriber to the callback. What I think we should do is model this as a pub/sub pattern. Let's chat about this, it may just be easier if we pair-program this bit.